118-pascals-triangle.py

```diff
@@ -0,0 +1,56 @@
+
+"""
+118. Pascal's Triangle
+Easy
+"""
+from leetcode_runner import LeetCode, TestCase, Args
+from typing import *
+
+PROBLEM = """
+Given an integer numRows, return the first numRows of Pascal's triangle.
+In Pascal's triangle, each number is the sum of the two numbers directly above it as shown:
+
+ 
+Example 1:
+Input: numRows = 5
+Output: [[1],[1,1],[1,2,1],[1,3,3,1],[1,4,6,4,1]]
+Example 2:
+Input: numRows = 1
+Output: [[1]]
+
+ 
+Constraints:
+
+1 <= numRows <= 30
+
+
+"""
+
+class Solution:
+    """
+    ACcepted
+    Time Complexity: O(n^2)
+    Space Complexity: O(n^2)
```

Comment:
Check space complexity

Reply:
Are you suggesting that the nested list to be returned should not be counted as auxiliary memory? And its O(1)